From 2cfc3b257b72bbf160f2ccb06a323d2b06a3404e Mon Sep 17 00:00:00 2001
From: Vaibhav Bedia <vaibhav.bedia@ti.com>
Date: Tue, 6 Mar 2012 12:12:07 +0530
Subject: [PATCH 0854/1092] ARM: OMAP: AM335x evm: Getting rid of memory
 accessors with barriers

We should be using readl/writel since they have barriers in place

Signed-off-by: Vaibhav Bedia <vaibhav.bedia@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index 00c0579..e18ef06 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -2201,8 +2201,8 @@ static int am335x_rtc_init(void)
 		return -ENOMEM;
 
 	/* Unlock the rtc's registers */
-	__raw_writel(0x83e70b13, base + 0x6c);
-	__raw_writel(0x95a4f1e0, base + 0x70);
+	writel(0x83e70b13, base + 0x6c);
+	writel(0x95a4f1e0, base + 0x70);
 
 	/*
 	 * Enable the 32K OSc
@@ -2210,7 +2210,7 @@ static int am335x_rtc_init(void)
 	 * Since we want the clock to be running before mmc init
 	 * we need to do it before the rtc probe happens
 	 */
-	__raw_writel(0x48, base + 0x54);
+	writel(0x48, base + 0x54);
 
 	iounmap(base);
 
-- 
1.7.11.2

